README.md

```diff
@@ -0,0 +1,6 @@
+# Repository forms
+
+[![Build Status](https://img.shields.io/travis/ezsystems/repository-forms.svg?style=flat-square)](https://travis-ci.org/ezsystems/repository-forms)
+[![Downloads](https://img.shields.io/packagist/dt/ezsystems/repository-forms.svg?style=flat-square)](https://packagist.org/packages/ezsystems/repository-forms)
+[![Latest stable](https://img.shields.io/packagist/v/ezsystems/repository-forms.svg?style=flat-square)](https://packagist.org/packages/ezsystems/repository-forms)
```

Comment:
Yes, `/v` is for all versions that are not marked as pre release. For pre releseases we can use `/vpre`.